bears/natural_language/LanguageToolBear.py

```diff
@@ -45,7 +46,8 @@ def run(self,
         :param natural_language:           A locale representing the language
                                            you want to have checked. If set to
                                            'auto' the language is guessed.
-                                           If the language cannot be guessed,
+                                           If the language cannot be guessed
+                                           or an invalid language is guessed,
```

Comment:
not "or an invalid language is provided"? I mean an invalid language can't be guessed, either the guess works or it fails with a `ValueError`, or not?

Reply:
Ya, so I am changing ``invalid language`` to ``unsupported language`` as the language is not supported by ``language_check``.